deny.toml

```diff
@@ -10,7 +8,6 @@ allow = [
     "LicenseRef-ring",
     "LicenseRef-webpki",
     "MIT",
-    "Unicode-DFS-2016",
```

Comment:
What's the story with this allowed license removal? Not in use?

Reply:
Yup!